metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/xml/SraEntryXmlParser.java

```diff
@@ -19,22 +19,23 @@
 package uk.ac.ebi.ampt2d.metadata.importer.xml;
 
 import org.apache.xmlbeans.XmlException;
-import org.w3c.dom.Document;
 import uk.ac.ebi.ampt2d.metadata.persistence.entities.ReferenceSequence;
 import uk.ac.ebi.ampt2d.metadata.persistence.entities.Taxonomy;
 
-import javax.xml.xpath.XPath;
-import javax.xml.xpath.XPathConstants;
 import java.util.Arrays;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-public class SraEntryXmlParser extends SraXmlParser<ReferenceSequence>  {
+public class SraEntryXmlParser extends SraXmlParser<ReferenceSequence> {
```

Comment:
Is `SraEntryXmlParser` still an appropriate name? If this is focused on `ReferenceSequence` objects...

Reply:
I just went with the root tag names of the Xmls, as we have assembly a reference sequence as well but it has SraAssemblyXml parser. but I can change to ReferenceSequenceParser anyways if you still feel so.

Reply:
I see, but I think we should still try to express more clearly the purpose of the class. What about `SraNonAssemblyReferenceXmlParser`?

Reply:
ok, I can rename it to SraNonAssemblyReferenceXmlParser that sounds rational.